components/script/dom/bindings/trace.rs

```diff
@@ -372,6 +373,7 @@ unsafe impl<A: JSTraceable, B: JSTraceable, C: JSTraceable> JSTraceable for (A,
 
 unsafe_no_jsmanaged_fields!(ActiveAttribInfo);
 unsafe_no_jsmanaged_fields!(ActiveUniformInfo);
+unsafe_no_jsmanaged_fields!(Box<dyn Fn(&Node) -> Box<dyn Iterator<Item = DomRoot<Node>>> + 'static>);
```

Comment:
Hmm, how is this safe at all?

Reply:
I had my huge doubts as well, and eventually I created a trait to encapsulate this idea instead.